pom.xml

```diff
@@ -170,6 +170,7 @@
             <plugin>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
+                <version>3.3.0</version>
```

Comment:
This is in line with: https://github.com/OpenHFT/Chronicle-Core/blob/master/pom.xml#L147